assets/components/FormSection.tsx

```diff
@@ -1,8 +1,14 @@
 import React, {useState} from 'react';
-import PropTypes from 'prop-types';
 
-export function FormSection({name, testId, children}) {
-    const [opened, setOpened] = useState(name == null);
+interface IProps {
+    initiallyOpen?: boolean;
+    name?: string;
+    testId: string;
```

Comment:
```
interface Test {
    'data-test-id'?: string;
}
```

Keep the same name for test IDs

Reply:
This name isn't convenient as you have to destructure props on another line in order to access it with `props['data-test-id']`